src/value.rs

```diff
@@ -113,6 +120,99 @@ impl<'gc> Value<'gc> {
         }
     }
 
+    // A small helper function to handle division-like zero handling
```

Comment:
Small nit, this doesn't need to be a method on Value, this and `copysign` can go to the bottom of the file since they're internal helper functions.